README.md

```diff
@@ -1,20 +1,18 @@
 # bat-status
 Check battery capacity and send desktop notification via `notify-send`
-if it gets low.
+if it reaches certain charge levels.
```

Comment:
how about:` if it falls below a certain charge level` ?

Reply:
I think falls is not the correct word to use here

I would got with reaches aswell.

Reply:
Perhaps `if it drops below…` is acceptable?

Reply:
`if it drops below a certain charge level`